modules/plugin/jdbc/jdbc-oracle/src/main/java/org/geotools/data/oracle/OracleNGDataStoreFactory.java

```diff
@@ -124,6 +124,15 @@ public class OracleNGDataStoreFactory extends JDBCDataStoreFactory {
                     "Specifies the timeout when negotiating a session with the database listener (milliseconds)",
                     false);
 
+    /** Specifies whether column REMARKS metadata will be returned. */
+    public static final Param GET_COLUMN_REMARKS =
+            new Param(
+                    "Get column remarks",
+                    Boolean.class,
+                    "Specifies whether column REMARKS metadata will be returned",
```

Comment:
I would be a bit more clear:

> Indicates whether column REMARKS metadata available for attribute description

Reply:
Why only column remarks --> attribute description? Why not table remarks --> feature type description?

Reply:
Description changed, as requested.  Table remarks was out of scope for this particular effort, but happy to do it in a separate PR.